go.sum

```diff
@@ -14,13 +16,18 @@ github.com/golang/mock v1.1.1/go.mod h1:oTYuIxOrZwtPieC+H1uAHpcLFnEyAGVDL/k47Jfb
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/golang/protobuf v1.3.2 h1:6nsPYzhq5kReh6QImI3k5qWzO4PEbvbIW2cwSfR/6xs=
 github.com/golang/protobuf v1.3.2/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
+github.com/golang/protobuf v1.3.3-0.20200128181022-d23c5127dc24 h1:iBOMsHeiWDdGDzkSvswDLJU9FNTmQYIlzBzI4/V0p0g=
```

Comment:
Now go.sum keeps those, too.

Regenerate? Or manually delete and then go mod tidy?

Reply:
Forgot to `go mod tidy`; travis discovered that too.

Manually editing `go.sum` is never a good idea as far as I can tell.  A subsequent `go mod tidy` will re-add what you deleted if it was necessary, or else running it first would have deleted it for you.  And you can't tell what is necessary because it includes all the transitive dependencies.